va/caa_test.go

```diff
@@ -909,7 +940,6 @@ func TestMultiCAARechecking(t *testing.T) {
 		{
 			name:                     "1 hijacked RVA, CAA issuewild type present, 1 failure allowed",
```

Comment:
Is it worth removing the "1 failure allowed" text from these test names, since that's now implicit in the number of RVAs present? Or better yet, prefixing every test name with the number of RVAs present?